Classes/IndexQueue/RecordMonitor.php

```diff
@@ -242,6 +243,19 @@ public function processDatamap_afterDatabaseOperations(
         $recordTable = $table;
         $recordUid = $uid;
 
+        // Check if record has already been processed since DataHandler sends processDatamap_afterDatabaseOperations
```

Comment:
We should move this code into an own method "isRecordAllreadyProcessed" which returns true or false and stepout when it returns true